net-irc/dccserver/dccserver-0.5-r2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="linux implementation of the mirc dccserver command"
+HOMEPAGE="https://www.nih.at/dccserver/"
+SRC_URI="https://www.nih.at/dccserver/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~riscv ~x86"
+
+BDEPEND="dev-libs/libbsd"
```

Comment:
This is DEPEND + RDEPEND

Reply:
I am not sure if I did this right. Can you have a look?